prusti/src/bin/prusti-driver.rs

```diff
@@ -57,6 +57,7 @@ fn main() {
     args.push("-Zdump-mir-dir=log/mir/".to_owned());
     args.push("-Zdump-mir=renumber".to_owned());
     args.push("-Zalways-encode-mir".to_owned());
+    args.push("-Zcrate-attr=feature(stmt_expr_attributes)".to_owned());
```

Comment:
Why is this feature strictly necessary?

Reply:
I suppose the reason why it works for spec-only closures without this feature is that spec-only closures never get used:
```Rust
#[prusti::spec_only]
#[prusti::expr_id = "34265d3e9c6942f1b164a73d3a866e2c_101"]
|| -> bool { a > b };
```
i.e. the attribute will be added to the closure here, because the statement consists only of the closure, whereas I need to do something with the closure, e.g.
```Rust
#[some_annotation]
let cl = || { ... };
```
meaning the annotation will be added to the _assignment_ rather than the closure (from my understanding; at least it's definitely not added to the closure, I've checked), and so the encoder won't find it.